themes/10up-block-theme/.lintstagedrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "*.css": [
+      "10up-toolkit lint-style"
+    ],
+    "*.js": [
+      "10up-toolkit lint-js"
+    ],
+    "*.jsx": [
+      "10up-toolkit lint-js"
+    ],
+    "*.php": [
+      "./vendor/bin/phpcs --extensions=php --warning-severity=8 -s"
```

Comment:
**suggestion:** these arguments should in the config itself to prevent inconsistencies with IDE integration and lintstagedrc

```
    <fileextensions>
        <extension>php</extension>
    </fileextensions>

    <!-- Set the severity level for warnings -->
    <arg name="warning-severity" value="8"/>

    <!-- Show sniff codes in output (-s option) -->
    <arg name="s"/>
```